examples-extra/todoapp/src/useTodos.tsx

```diff
@@ -1,129 +1,66 @@
-import { ActionValidationError } from "@osdk/client";
+import { ActionValidationError, type Osdk } from "@osdk/client";
 import { useCallback, useEffect } from "react";
-import useSWR from "swr";
-import { foundryClient, foundryClient2 } from "./foundryClient";
-import { isOk, type Result, ReturnEditsMode } from "./generatedNoCheck";
-import type { Todo } from "./generatedNoCheck/ontology/objects";
+import useSWR, { KeyedMutator } from "swr";
+import { $ } from "./foundryClient";
 import * as MyOsdk from "./generatedNoCheck2";
-
-function orThrow<T, E>(result: Result<T, E>) {
-  if (isOk(result)) {
-    return result.value;
-  } else {
-    throw result.error;
+import { ObjectTypeDefinition } from "@osdk/api";
+
+declare global {
+  interface ArrayConstructor {
+    /**
+     * Creates an array from an async iterator or iterable object.
+     * @param iterableOrArrayLike An async iterator or array-like object to convert to an array.
+     */
+    fromAsync<T>(
```

Comment:
isn't fromAsync a built in feature? Why do we need to redeclare this here?

Reply:
Its not in the types yet, so while we can count on modern browsers having it and polyfills taking care of it, it was only merged a few weeks ago into ts's main branch